uaclient/system.py

```diff
@@ -181,8 +190,10 @@ def get_platform_info() -> Dict[str, str]:
             "series": series.lower(),
             "kernel": get_kernel_info().uname_release,
             "arch": get_dpkg_arch(),
+            "host_type": get_host_type(),
```

Comment:
I'm having second thoughts. Perhaps we should call this `virt` or something like that? It would be weird to have "hostType: none".

Reply:
`virt` makes sense to me

Reply:
Changing it to `virt` then